src/components/config_profile/test/profile_test.cc

```diff
@@ -724,6 +724,193 @@ TEST_F(ProfileTest, CheckIntContainerInSecurityData) {
 }
 #endif
 
+TEST_F(ProfileTest, CheckSdlVersion) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Declare empty string
+  std::string sdl_version;
+  // Check initial sdl_version
+  EXPECT_EQ(sdl_version, profile_.sdl_version());
+}
+
+TEST_F(ProfileTest, CheckLinkToWebHmi) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
```

Comment:
@Ypostolov Looks that 2 lines above are repeated in all test cases. You can move them to the constructor of the test or SetUp method.

Reply:
@AByzhynar 
fixed in https://github.com/smartdevicelink/sdl_core/pull/1995/commits/8b04ba878df1f894f8688d2196f7a6c4b29b9328